ramls/scheduled-notice-storage.raml

```diff
@@ -0,0 +1,53 @@
+#%RAML 1.0
+title: Scheduled Notice Storage
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Scheduled Notice Storage API
+    content: <b>Storage for scheduled notices</b>
+
+types:
+  scheduled-notice: !include scheduled-notice.json
+  scheduled-notices: !include scheduled-notices.json
+  errors: !include raml-util/schemas/errors.schema
+  parameters: !include raml-util/schemas/parameters.schema
+
+traits:
+  language: !include raml-util/traits/language.raml
+  pageable: !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+  validate: !include raml-util/traits/validation.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/scheduled-notice-storage:
```

Comment:
I think both the post and put endpoints need to declare the `validate` trait, as they can respond with validation errors.

For [example](https://github.com/folio-org/mod-inventory-storage/blob/b8e5b4f13594ce22fede6e1e4c1a8a92b6508834/ramls/location.raml#L43)

Maybe, in future, the standard resource type will be changed to include this trait all of the time

Reply:
Thank you. I thought the standard resource type includes validate trait.

Added the trait for post and put.

Reply:
Thanks